src/lv_conf_kconfig.h

```diff
@@ -19,6 +19,10 @@ extern "C" {
 #include <nuttx/config.h>
 #endif
 
+#ifdef __ZEPHYR__
+#include <lv_conf_zephyr.h>
+#endif
+
```

Comment:
Actually not sure if this is the right approach as a new `ifdef` needs to be added for every RTOS/Platform (now there are already 3; ESP, NuttX & Zephyr)

Maybe this could be a solution (untested/pseudo code) and replace lines 13-25:
```
#ifdef LV_CONF_KCONFIG_EXTERNAL_INCLUDE
#include LV_CONF_KCONFIG_EXTERNAL_INCLUDE
#elif defined __has_include
#  if __has_include("lv_conf_kconfig_external.h")
#   include <lv_conf_kconfig_external.h>
#  endif
#endif
```

Then ESP, NuttX and Zephyr can add either an include path that contains `lv_conf_kconfig_external.h` or pass a global define `LV_CONF_KCONFIG_EXTERNAL_INCLUDE` to the compiler in case `__has_include` is not supported or an alternate include file name should be used.

With proposed code above if a new RTOS/Platform pops up the LVGL code base does not need changed.

What do you think?

cc: @kisvegabor  @v01d

Reply:
I think this is OK. The include with a macro is portable?

Reply:
With `LV_CONF_KCONFIG_EXTERNAL_INCLUDE` we rely on the RTOS to handle this on their side and if the RTOS has a slow release cycle, or they are not interested in officially adding LVGL related configs into their build, users won't be able to use LVGL with that RTOS. 

Although I can imagine this define in addition to the hardcoded includes. If `LV_CONF_KCONFIG_EXTERNAL_INCLUDE` is not set, try the hardcoded ones. 

What do you think?